src/ossia-max/src/ossia-max.hpp

```diff
@@ -86,9 +86,9 @@ class ossia_max
 {
 public:
   static ossia_max& instance();
-  static ossia::net::generic_device* get_default_device()
+  static std::shared_ptr<ossia::net::generic_device> get_default_device()
```

Comment:
could be worth returning a const&, shared_ptr can be somewhat slow to copy

Reply:
done in ceb3a6b7